open-metadata-distribution/open-metadata-assemblies/src/main/assemblies/egeria-omag.xml

```diff
@@ -74,6 +74,7 @@
                 <include>org.odpi.egeria:kafka-open-metadata-topic-connector:*</include>
                 <include>org.odpi.egeria:omrs-rest-repository-connector:*</include>
                 <include>org.odpi.egeria:open-lineage-janus-connector:*</include>
+                <include>org.odpi.egeria:search-integrator-api:*</include>
```

Comment:
This section of the assembly is for additional files that will be added into server/lib. I think you meant the integration connector rather than the api module?

This will also neeed to be added to the assembly that gradle builds in build.gradle - same directory.

There are two sections - similar idea to the pom.xml
a) Dependencies - just add that project in there
b) under 'distributions' a whole lot of sections by directory - so again add in the things you want into the same place as other connectors etc 

We don't actually use the gradle assembly, and it will need to be verified against maven, but it would be useful to try and keep in sync